packages/app-desktop/gui/ConfigScreen/ConfigScreen.search.test.ts

```
@@ -0,0 +1,45 @@
+import { SettingItem } from '@joplin/lib/models/Setting';
+import { settingMatchesSearch } from './configScreenUtils';
```

ESLint

[error] Expected linebreaks to be 'LF' but found 'CRLF'.

(linebreak-style)